### PR TITLE
Moved test_utils.py to utils/test.py

### PR DESCRIPTION
**Pull request by jackl-xilinx**

### CODE REVIEW

File: python/utils/ml.py
Comment (fifield) [inline python/utils/ml.py:1]: needs license/copyright

File: python/utils/test.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# test_utils.py -*- Python -*-
+#
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: MIT
```
Comment (fifield) [inline python/utils/test.py:4]: Should be Apache, not MIT

File: python/utils/xrt.py
Comment (fifield) [inline python/utils/xrt.py:1]: needs license/copyright

File: python/utils/xrt.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# from npu.runtime
```
Comment (fifield) [inline python/utils/xrt.py:1]: needs license/copyright

**Review by fifield — CHANGES_REQUESTED** [review]

**Comment by jgmelber** [comment]
@fifield can you check if this is ready to merge?

**Review by fifield — APPROVED** [review]

**Pull request merged**